src/polymer-language-server.ts

```diff
@@ -106,12 +106,46 @@ connection.onInitialize((params): InitializeResult => {
   };
 });
 
+async function mapAllInitialDocuments(editorService: EditorService) {

```

Comment:
Why does this function take an editorService parameter, but other functions use the module scoped variable?